src/main/java/edu/phema/elm_to_omop/vocabulary/phema/PhemaConceptSet.java

```diff
@@ -12,6 +13,7 @@
  * name, which would obviate the need for this
  * class.
  */
+@JsonIgnoreProperties(value = {"circeConceptSet"})
```

Comment:
This is necessary because we had a `getCirceConceptSet()` method, which Jackson took to mean that we wanted a `circeConceptSet` in the serialized JSON 🙂. The method is actually just there to convert our custom concept set to a Circe concept set (by dropping the `oid` field).